src/app/components/views/weather-view/weather-view.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="container">
```

Comment:
this file has spacing of 4...please have tabs/indents be 2 spaces

Reply:
should be fixed now